geniza/corpus/templates/corpus/snippets/document_tabs.html

```diff
@@ -11,7 +11,7 @@
         {% with n_links=document.fragment_urls|length %}
         {# Translators: n_links is number of external links #}
         {% blocktranslate asvar elink_text %}External Links ({{ n_links }}){% endblocktranslate %}
-        <li>{% if n_links > 0 %}<a>{{ elink_text }}</a>{% else %}<span>{{ elink_text }}</span>{% endif %}</li>
+        <li>{% if n_links > 0 %}<a href="#">{{ elink_text }}</a>{% else %}<span>{{ elink_text }}</span>{% endif %}</li>
```

Comment:
do we need to do anything to disable this for MVP or do you think this implementation is fine?

Reply:
probably we should just comment out the whole thing, I guess? that's a good point